pkg/license/multichannel.go

```diff
@@ -40,11 +43,20 @@ func VerifyAndUpdateLicense(log *logger.CLILogger, license *kotsv1beta1.License,
 		return nil, nil
 	}
 	if isAirgap {
+		log.ActionWithSpinner("Verifying if channel slug %q is allowed by the license", preferredChannelSlug)
 		if !canInstallFromChannel(preferredChannelSlug, license) {
-			return nil, errors.New("requested channel not found in supplied license")
+			log.FinishSpinnerWithError()
+			return license, errors.New(fmt.Sprintf("channel slug %q is not allowed by license", preferredChannelSlug))
+		}
+		log.FinishSpinner()
+		validChannels := []string{}
+		for _, channel := range license.Spec.Channels {
+			validChannels = append(validChannels, fmt.Sprintf("%s/%s", license.Spec.AppSlug, channel.ChannelSlug))
 		}
+		log.ChildActionWithoutSpinner(fmt.Sprintf("To install an allowed channel, use one of the following: %s", strings.Join(validChannels, ", ")))
```

Comment:
Isn't the license / channel installable at this point since it got past the `canInstallFromChannel` check?